physics/smoke_dust/rrfs_smoke_wrapper.meta

```diff
@@ -810,6 +869,30 @@
   type = real
   kind = kind_phys
   intent = inout
+[peak_hr_out]
+ standard_name = peak_hr_fire
+ long_name = hour of peak fire emissions
+ units = none
+ dimensions = (horizontal_loop_extent)
+ type = real
+ kind = kind_phys
+ intent = out
+[lu_nofire_out]
+ standard_name = lu_nofire_out
```

Comment:
Between the standard name and long name, I still don't really understand what this variable represents. Can we clear this up and come up with a better standard name? (Same for the variable below)

Reply:
@grantfirl The standard names will be updated to "sum_of_land_use_fractions_for_no_fire_pixels" and "sum_of_land_use_fractions_for_cropland_fire_pixels". Thanks.